models/DataObject/ClassDefinition/Data/EncryptedField.php

```diff
@@ -513,4 +493,14 @@ public function denormalize(mixed $value, array $params = []): Model\DataObject\
 
         return $value;
     }
+
+    public function getColumnType(): array|string|null
```

Comment:
```suggestion
    public function getColumnType(): string
```
We could narrow down the return types to either string or array for all of the `getColumnType` and `getQueryColumnType` methods. Like it is already the case here https://github.com/pimcore/pimcore/blob/a2bc2e66e92b521d2bd7ef3cc6765c8eec715389/models/DataObject/ClassDefinition/Data/NumericRange.php#L166 (we can even omit the `null`)

I think we should be consistent at least.